scripts/lassen-parallel-spawner.sh

```diff
@@ -2,8 +2,8 @@
 #
 # Used to wrap the spawning of parallel mirgecom drivers on Lassen
 # unset CUDA_CACHE_DISABLE
-POCL_CACHE_ROOT=${POCL_CACHE_ROOT:-"/tmp/$USER/pocl-scratch"}
-XDG_CACHE_ROOT=${XDG_CACHE_HOME:-"/tmp/$USER/xdg-scratch"}
+POCL_CACHE_ROOT=${POCL_CACHE_ROOT:-"/usr/workspace/wsa/$USER/pocl-scratch"}
+XDG_CACHE_ROOT=${XDG_CACHE_HOME:-"/usr/workspace/wsa/$USER/xdg-scratch"}
```

Comment:
I dislike this.  The default setting in /tmp is better than setting it to something some folks may not have access to.   It would be better, imo, to have the user set something that overrides POCL_CACHE_ROOT/XDG_CACHE_ROOT if they don't want node-local caching.  

If we want to invert that and make it so that the user must specify when they _want_ node-local caching, then that'd be ok too, but it should then just default to the CWD of the job or something like that, not a file path.